Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py

```diff
@@ -312,19 +315,31 @@ def main():
     params = demisto.params()
 
     LOG(f'Command being called is {demisto.command()}')
+
     try:
+        self_deployed = params.get('self_deployed', False)
```

Comment:
id this did not have self deployed app till now, it will not be needed to be cover in the readme?

Reply:
There was a `self_deploy`, but I moved its init up so I can check it.